atlp-rwanda/e-commerce-lydia-32-bn
# Ft public chat 187570895 Users should be able to chat on the App

### What does this PR do?

This PR introduces a public chat functionality to the application, allowing users to engage in real-time conversations, ask questions, and share information with other online users.

### Description of Task to be completed?

The main tasks involved in implementing the public chat feature are:

1. Create a chat room or forum where users can post messages and questions.
2. Implement real-time messaging capabilities, ensuring that new messages are instantly visible to all online users.
3. Display previous chat messages when a user opens the chat tab.

How should this be manually tested?

To manually test the public chat functionality, follow these steps:

1. Open the application and ensure you are logged in as a user.
2. Navigate to the chat route of the application.
3. Verify that you can see any previously posted chat messages.
4. Compose a new chat on the new route for posting message and send it.
5. Ensure that your message is immediately visible in the chat room.


### What are the relevant pivotal tracker stories

https://www.pivotaltracker.com/n/projects/2703469
